web-server/src/components/Teams/useTeamsConfig.tsx

```diff
@@ -498,26 +499,26 @@ const useReposSearch = () => {
     async (query) => {
       depFn(isLoading.true);
       // cancel the previous request if it exists
-      if (cancelTokenSource || !query) {
-        cancelTokenSource.cancel('Operation canceled due to new request.');
+      if (controllerRef.current) {
```

Comment:
This seems to change the condition here. Intentional?

Reply:
In the original if statement the cancelTokenSource was always true regardless of the query. 

(+if you can !cancelTokenSource, but !query then it would throw en error that _function cancel is not found on undefined._)  

So the `|| !query` is no needed there.